docs-starlight/src/content/docs/02-features/05-run-queue.mdx

```diff
@@ -24,57 +24,247 @@ Terragrunt analyzes these dependencies to determine the correct order of operati
 
 ### Example DAG
 
-Consider a setup where Unit B depends on Unit A, and Unit C depends on Unit B. Unit D has no dependencies.
+Consider a setup where:
+
+- Unit "dependent" depends on unit "dependency".
+- Unit "dependency" depends on unit "ancestor-dependency".
```

Comment:
Maybe add that this is an "external dependency"?

Reply:
This is something I occasionally get confused about in how the CLI works, so I might need your help finding a good way to explain this.

An "external dependency" is something that's related to the current working directory. You can't determine whether something is an external dependency without discussing _where_ users are running Terragrunt. If a user is running Terragrunt at the root of the project repository, then nothing is technically an external dependency.